showcase/showcase-components/showcase-dropdown.js

```diff
@@ -18,24 +18,30 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import ReactDOM from 'react-dom';
-import React, {Component} from 'react';
-import document from 'global/document';
+import React from 'react';
+import PropTypes from 'prop-types';
 
-import ComplexChartExample from './complex-chart-example';
-import '../../src/styles/examples.scss';
-
-export default class App extends Component {
+class ShowcaseDropdown extends React.Component {
+  state = {
+    open: false
+  }
   render() {
+    const {items} = this.props;
+    const {open} = this.state;
     return (
-      <article>
-        <h1>Simple Complex Chart Example</h1>
-        <section>
-          <ComplexChartExample />
-        </section>
-      </article>
+      <div className="dropdown-wrapper">
+        <div className="dropdown-button" onClick={() => this.setState({open: !open})}>
```

Comment:
maybe create a class property function to toggle the open status so it won't create two functions on every render (micro opti Ino)